code/modules/mob/say.dm

```diff
@@ -67,13 +83,15 @@
 //parses the message mode code (e.g. :h, :w) from text, such as that supplied to say.
 //returns the message mode string or null for no message mode.
 //standard mode is the mode returned for the special ';' radio code.
-/mob/proc/parse_message_mode(var/message)
-	if(length(message) >= 1)
-		if(copytext_char(message,1,2) == get_prefix_key(/decl/prefix/radio_main_channel))
-			return MESSAGE_MODE_DEFAULT
-		if(length(message) >= 2)
-			var/channel_prefix = copytext_char(message, 1, 3)
-			return department_radio_keys[channel_prefix]
+/mob/proc/parse_message_mode(var/message, var/standard_mode=MESSAGE_MODE_DEFAULT)
+	if(length(message) <= 0)
+		return null
+	var/initial_char = copytext_char(message,1,2)
```

Comment:
maybe lowertext it too, since list doesn't hold uppercase versions anymore?

Reply:
I'm pretty sure the intent is to have exactly one key for a channel, so that :e and :E would go to different channels. Doubles the number of keys available. Does anyone actually use the uppercase letters for the built-in channels?

Reply:
Hulkified people do